python/py-canmatrix/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        ebroecker canmatrix 0.6
+
+name                py-canmatrix
+categories-append   devel
+platforms           darwin
+license             BSD
+maintainers         {@ierofant gmail.com:aw.kornilov} openmaintainer
+description         Converting Can (Controller Area Network) Database Formats .arxml .dbc .dbf .kcd
+
+long_description    Canmatrix implements a "Python Can Matrix Object" which describes the can-communication \
+                    and the needed objects (Boardunits, Frames, Signals, Values, ...) \
+                    Canmatrix also includes two Tools (canconvert and cancompare) for converting and comparing \
+                    CAN databases.
+
+python.versions     27 34 35 36 37
+
+checksums           rmd160  a73bf7be09a7ede94e4301b868e56d79b7b98650 \
+                    sha256  086303b0ef4d3da4eeb86b61813f60b902254945ea4a44cf833b10c0ab7cfafe \
+                    size    832836
+
+if {${name} ne ${subport}} {
+    depends_build-append    port:py${python.version}-setuptools
```

Comment:
as @pmetzger said normally the livecheck is turned off for subports. You can do so by adding ```    livecheck.type      none``` within the if-statement.

Reply:
Done